File: ViewersAndVisualizers/ViewersAndVisualizersExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ViewersAndVisualizers
	<TestClass, TestFixture> _
	Public Class ViewersAndVisualizersExamples
        <TestMethod(), Test(), Owner("WinForm")> _
	        Public Sub DocumentExplorer()
	            TestHelper.SetDataDir("ViewersAndVisualizers/DocumentExplorer")
	
	            DocumentExplorerExample.Program.Main()
        End Sub

	End Class
End Namespace

File: Examples/ViewersAndVisualizers/ViewersAndVisualizersExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ViewersAndVisualizers
{
    [TestClass, TestFixture]
    public class ViewersAndVisualizersExamples
    {	
        [TestMethod, Test, Owner("WinForm")]
        public void DocumentExplorer()
        {
            TestHelper.SetDataDir("ViewersAndVisualizers/DocumentExplorer");

            DocumentExplorerExample.MainForm.Main();
        }

    }
}
